package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cloudflare-router",
-  "version": "0.1.0",
+  "version": "0.2.25",
```

```suggestion
  "version": "0.2.0",
```

the patch version was just for our testing right?